pdns/README-dnsdist.md

```diff
@@ -130,28 +130,28 @@ To force a server down, try:
 ```
 
 The 'DOWN' in all caps means it was forced down. A lower case 'down'
-would've meant that `dnsdist` itself had concluded the server was down.
+would have meant that `dnsdist` itself had concluded the server was down.
 Similarly, setUp() forces a server to be up, and setAuto() returns it to the
-default availability-probing.
+default availability probing.
 
 To change the QPS for a server:
 ```
 > getServer(0):setQPS(1000)
 ```
 
 By default, the availability of a downstream server is checked by regularly
-sending an A query for "a.root-servers.net.". A different query type and target
+sending an A query for `a.root-servers.net.`. A different query type and target
 can be specified by passing, respectively, the `checkType` and `checkName`
 parameters to `newServer`. The default behavior is to consider any valid response
-with a RCODE different from ServFail as valid. If the `mustResolve` parameter
+with an RCODE different from ServFail as valid. If the `mustResolve` parameter
 of `newServer` is set to true, a response will only be considered valid if
 its RCODE differs from NXDomain, ServFail and Refused.
 The number of health check failures before a server is considered down is
-configurable via the`maxCheckFailures` parameter, defaulting to 1.
+configurable via the `maxCheckFailures` parameter, defaulting to 1.
 The `CD` flag can be set on the query by setting `setCD` to true.
 
 ```
-newServer({address="192.0.2.1", checkType="AAAA", checkName="a.root-servers.net.", mustResolve=true})
+newServer({address="192.0.2.1", checkType="AAAA", checkName="a.root-servers.net.", mustResolve=true}) -- FIXME: what is this code segment for?
```

Comment:
I guess the intent was to document the `checkType`, `checkName` and `mustResolve` parameters described above